web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -113,15 +125,18 @@ const TeamReviews = ({ onBack, periodId }) => {
   const history = useHistory();
   const location = useLocation();
 
+  const [approvalMode, setApprovalMode] = useState(false);
   const [assignments, setAssignments] = useState([]);
   const [confirmOpen, setConfirmOpen] = useState(false);
   const [memberSelectorOpen, setMemberSelectorOpen] = useState(false);
+  const [nameQuery, setNameQuery] = useState('');
```

Comment:
This is set when the user enters part of a name to filter the members that are dlsplayed in the list at the bottom of the page.